libs/network/include/network/service/protocol.hpp

```diff
@@ -161,106 +142,6 @@ class Protocol
   virtual void ConnectionDropped(connection_handle_type /*connection_handle*/)
```

Comment:
Do we still have connection handles and would it not be more appropriate to use context/address?

Reply:
Nope we still have connection handles. This class needs a little more work